frontend/package.json

```diff
@@ -108,6 +110,8 @@
     "jsdom": "^16.6.0",
     "meow": "^10.0.1",
     "globby": "^11.0.3",
+    "next": "^11.1.0",
+    "node-gyp": "^8.1.0",
```

Comment:
What is this required for?

Reply:
I was able to remove this and most of the other audit resolutions too, after next upgrade